### PR TITLE
Storage blob support

### PR DESCRIPTION
**Pull request by abner-dou**
updated `azurestack_storage_blob` resource

### CODE REVIEW

File: internal/services/storage/registration.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ func (r Registration) SupportedDataSources() map[string]*pluginsdk.Resource {
 func (r Registration) SupportedResources() map[string]*pluginsdk.Resource {
 	return map[string]*pluginsdk.Resource{
 		"azurestack_storage_account":   storageAccount(),
-		"azurestack_storage_blob":      storageBlob(),
+		"azurestack_storage_blob":      resourceStorageBlob(),
```
Comment (katbyte) [inline internal/services/storage/registration.go:32]: could we make this consistant?
```suggestion
		"azurestack_storage_blob":      storageBlob(),
```

all function names should have resource removed like this

**Review by katbyte — CHANGES_REQUESTED** [review]

File: internal/services/storage/registration.go (hunk 1 above)
Comment (abner-dou) [reply internal/services/storage/registration.go]: done :D

File: internal/services/storage/storage_blob_resource.go [hunk 2]
```diff
@@ -8,63 +8,54 @@ import (
...
 
-func storageBlob() *schema.Resource {
+func resourceStorageBlob() *schema.Resource {
 	return &schema.Resource{
-		Create: storageBlobCreate,
-		Read:   storageBlobRead,
-		Delete: storageBlobDelete,
-
-		// TODO check schema and confirm old stack provider can upgrade to this
+		Create:        resourceStorageBlobCreate,
```
[the remark this answers is not in the document]
Comment (abner-dou) [reply internal/services/storage/storage_blob_resource.go]: done :)

File: internal/services/storage/storage_blob_resource.go [hunk 3]
```diff
@@ -75,22 +66,28 @@ func storageBlob() *schema.Resource {
 					"Page",
 				}, false),
 			},
-
```
[the remark this answers is not in the document]
Comment (abner-dou) [reply internal/services/storage/storage_blob_resource.go]: done :)

File: internal/services/storage/storage_blob_resource_test.go [hunk 4]
```diff
@@ -479,6 +654,78 @@ resource "azurestack_storage_blob" "test" {
...
 }
 
+func (r StorageBlobResource) contentMd5ForLocalFile(data acceptance.TestData, fileName string) string {
+	template := r.template(data, "blob")
+	return fmt.Sprintf(`
+%s
+provider "azurestack" {
+  features {}
+}
+resource "azurestack_storage_blob" "test" {
```
[the remark this answers is not in the document]
Comment (abner-dou) [reply internal/services/storage/storage_blob_resource_test.go]: done :)

**Review by katbyte — APPROVED** [review]
LGTM 🦀

**Pull request merged**